### ISSUE TITLE
LineBox without corners broken

### ISSUE THREAD

**Issue by fabi321** [urwid/urwid#420]
##### Description:
if you use a LineBox with only left or right corners as empty string, the corners get rendered as space

##### Affected versions (if applicable)
- [x] `master` branch (56194f9dd7f9288361c37bbc694f18762c5158d7)
- [x] Latest stable version from `pypi`
- [ ] Other (specify source)

##### Steps to reproduce (if applicable)
```
urwid.MainLoop(urwid.LineBox(urwid.SolidFill(), tlcorner='', blcorner='', lline='')).run()
```
has spaces on the top left corner and bottom left corner.

##### Expected/actual outcome
No spaces.

**Comment by penguinolog** [comment]
I'm a bit disagree that it's a bug: In some cases it's needed by design to have empty corners.
Setting corners explicit for my opinion is better than implicit.

**Comment by penguinolog** [comment]
Issue fixed in #748
